server/index.js

```diff
@@ -1,21 +1,25 @@
-const express = require('express');
-const axios = require('axios');
-const morgan = require('morgan');
-const {TOKEN} = require('../config.js');
+const express = require("express");
+const axios = require("axios");
+const morgan = require("morgan");
+const { TOKEN } = require("../config.js");
 
-let app = express();
-var router = require('./routes.js');
-app.use('/catwalk', router); //placeholder
+// connect to mongodb use mongoose
+const mongoose = require("mongoose");
+mongoose.connect("mongodb://localhost:27017/shop");
```

Comment:
I would suggest move line 6-8 to a different file, so we have a consistent format.